### ISSUE TITLE
Scene Thumbnail Generation Error

### ISSUE THREAD

**Issue by TheDuriel** [godotengine/godot#40467]
**Godot version:**

3.2 Stable

**OS/device including version:**

Win 10

**Issue description:**

When you save a Scene an error concerning its thumbnail is thrown if:

The Scene contains no visible nodes.

The Scene sizes visible contents exceed 16,384 x 16,384 Pixels area.

**Steps to reproduce:**
See above. Saving a Scene with a simple Node as root should suffice to trigger the message.

**Minimal reproduction project:**

**Comment by akien-mga** [comment]
>  See above. Saving a Scene with a simple Node as root should suffice to trigger the message.

Well more info will be needed, as I can't reproduce it.

I zoomed out to 100,000x80,000 and made sure to have Sprites in each corner, but I don't reproduce the issue either in Godot 3.2-stable or 3.2.2-stable. I tried making the canvas size 80,000x50,000 too to be sure (fair warning, it's slow).

![Screenshot_20200721_155618](https://user-images.githubusercontent.com/4701338/88063793-bc0a8780-cb6a-11ea-9f05-aebef0c927d5.png)
